scripts/alert_emails/campaign-monitor.ts

```diff
@@ -55,18 +61,20 @@ class CampaignMonitor {
     });
   }
 
-  async sendClassicEmail(data: EmailSendData): Promise<AxiosResponse> {
+  async sendClassicEmail(
+    data: EmailSendData,
+  ): Promise<AxiosResponse<CampaignMonitorMessageSent[]>> {
```

Comment:
nit: Returning an AxiosResponse leaks the implementation details of this function.  Would it make sense to just return `Promise<CampaignMonitorMessageSent>` and return `res.data`?

Feel free to punt.